packages/arch/packages/alert/README.md

```diff
@@ -142,10 +142,14 @@ An alert that is full width; removes border and border radius.
 
 ## License
 
-[MIT](./LICENSE) &copy; [Thinkmill](https://www.thinkmill.com.au/)
+[MIT](./LICENSE) © [Thinkmill](https://www.thinkmill.com.au/)
```

Comment:
Enabling `commonmark` makes the output compatible with the commonmark spec. The spec has a section on handling [entities and numeric character references](https://spec.commonmark.org/0.29/#entity-and-numeric-character-references)

Reply:
I prefer this anyway :)